### PR TITLE
Refactor/rm v1beta ingress

### PR DESCRIPTION
**Pull request by szuecs**
remove ingress v1beta1 and change all tests config accordingly

Missing: docs/ change to change all examples to v1 ingress

### CODE REVIEW

File: config/config.go [hunk 1]
```diff
@@ -144,14 +144,15 @@ type Config struct {
 
 	RefusePayload multiFlag `yaml:"refuse-payload"`
 
+	// KubernetesIngressV1 setting is deprecated and has no effect, only v1 is supported
+	KubernetesIngressV1 bool `yaml:"kubernetes-ingress-v1"`
```
Comment (AlexanderYastrebov) [inline config/config.go:148]: I think we can just remove it

Comment (szuecs) [reply config/config.go]: done in all the places (skipper, config and dataclient Options)

**Comment by szuecs** [comment]
I dropped rest of the references @AlexanderYastrebov please review.
One open question: Should I move all code from `dataclients/kubernetes/ingressv1.go` to `dataclients/kubernetes/ingress.go` and rename the ugly V1 Suffixes everywhere?

**Comment by AlexanderYastrebov** [comment]
> Should I move all code from dataclients/kubernetes/ingressv1.go to dataclients/kubernetes/ingress.go and rename the ugly V1 Suffixes everywhere?

Not sure, but if yes it is better done in a separate changeset.

**Comment by szuecs** [comment]
:+1:

**Comment by AlexanderYastrebov** [comment]
:+1:

**Comment by AlexanderYastrebov** [comment]
:+1:

**Comment by szuecs** [comment]
:+1:

**Pull request merged**